CONTRIBUTING.md

```diff
@@ -155,7 +167,7 @@ systemProp.test.couch.http=[http|https] # default 5984
 systemProp.test.couch.ignore.compaction=[true|false] # default false
 systemProp.test.couch.auth.headers=[true|false] # default false
 ```
-Note: some tests need to be ignored based on the configuration of the CouchDB instance you are using or if you are running against Cloudant. For example if the compaction endpoint is unavailable eg returns a 403 response, these tests should be disabled. 
+Note: some tests need to be ignored based on the configuration of the CouchDB instance you are using or if you are running against Cloudant. For example if the compaction endpoint is unavailable eg returns a 403 response, these tests should be disabled.
```

Comment:
I would say this is a contentious subject: Chicago style manual appears to agree with you whereas the gov.uk manual disagrees (https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#eg-etc-ie). Those guides also recommend alternatives like "that is" or "for example".